src/affiliate-manager/service/affManagerService.js

```diff
@@ -0,0 +1,187 @@
+import { inject } from "aurelia-framework";
+import { HttpClient } from "aurelia-fetch-client";
+import { Store } from "aurelia-store";
+import { setPartyId } from "../../store/store";
+
+@inject(HttpClient, Store)
+export class AffManagerService {
+
+  constructor(httpClient, store) {
+    this.httpClient = httpClient;
+    this.httpClient.configure(config => {
+        config
+          .withBaseUrl('https://localhost:8443/api/')
```

Comment:
I assume this is temporary? since in production proxy is set up, so baseUrls should look like
`api/v1/projects`. (without localhost)
nginx reverse proxy will route this request to BE. I also recommend using proxy locally.